lego/apps/articles/models.py

```diff
@@ -1,13 +1,18 @@
 from django.conf import settings
+from django.db.models import URLField
+from django.core.validators import URLValidator
 
 from lego.apps.content.models import Content
 from lego.apps.files.models import FileField
 from lego.apps.permissions.models import ObjectPermissionsModel
 from lego.utils.models import BasisModel
 
+from rest_framework.exceptions import ValidationError
+
 
 class Article(Content, BasisModel, ObjectPermissionsModel):
     cover = FileField(related_name="article_covers")
+    youtube_url = URLField(default="")
```

Comment:
You should add a custom validator here to check that the URL is for Youtube. See the `validators.py` file for inspiration